raiden/transfer/architecture.py

```diff
@@ -402,3 +402,30 @@ def token_network_address(self) -> TokenNetworkAddress:
     @property
     def channel_identifier(self) -> ChannelID:
         return self.canonical_identifier.channel_identifier
+
+
+class SuccessOrError:
+    """Helper class to be used when you want to test a boolean
+
+    and also collect feedback when the test fails. Initialize with any
+    number of "error message" strings. The object will be considered
+    truthy if there are no error messages.
+    """
+
+    def __init__(self, *error_messages: str):
```

Comment:
This will also need return type of `None`. I know it's stupid but if it does not have it mypy does not pick it up.